packages/fxa-payments-server/README.md

```diff
@@ -84,6 +84,10 @@ We use rescripts and yarn workspaces to manage our node packages. If you are enc
 
 This launch config currently only works for the tests under the `/src` directory in the `fxa-payments-server` package, not the `/server` directory.
 
+### L10N
+
+Strings are automatically extracted to the [`fxa-content-server-l10n` repo](https://github.com/mozilla/fxa-content-server-l10n) where they reach Pontoon for translations to occur by our l10n team and contributors. This is achieved by concatenating all of our .ftl (Fluent) files into a single `payments.ftl` file with the `merge-ftl` grunttask, and the `extract-and-pull-request.sh` script that runs in `fxa-content-server-l10n` on a weekly cadence. For more detailed information, check out the [ecosystem platform l10n](http://localhost:3000/ecosystem-platform/reference/localization) doc.
```

Comment:
I know we have this in `auth-server` too, but `extract-and-pull-request.sh` is actually no longer a script, it's a GH action. We do have `extract_strings.sh` over there though... I'll go through our docs later when we do doc tickets. 🤷‍♀️

Reply:
Should I entirely remove this bit?

> and the `extract-and-pull-request.sh` script that runs in `fxa-content-server-l10n` on a weekly cadence.

Reply:
Eh, either way actually since I guess a GH action is technically a script and this'll be tweaked in other tickets anyway, I just think of `.sh` files when I see `script`.

Oh, we can update that link though. This should go to the deployed ecosystem-platform docs, not `localhost:3000` (this was probably my mistake in auth-server's readme, lol).

Reply:
I removed the script name and updated the link